src/pages/publication/new_publication/LinkPublication.tsx

```diff
@@ -34,16 +33,30 @@ const LinkPublicationPanel: FC<LinkPublicationPanelProps> = ({ expanded, onChang
   const { t } = useTranslation();
   const [doiUrl, setDoiUrl] = useState('');
   const [doiTitle, setDoiTitle] = useState('');
+  const [loading, setLoading] = useState(false);
+  const [noHit, setNoHit] = useState(false);
   const dispatch = useDispatch();
-  const user = useSelector((state: RootStore) => state.user);
 
-  const handleConfirm = () => {
-    dispatch(createNewPublicationFromDoi(doiUrl, user.id, dispatch));
+  const createPublication = async () => {
+    // eslint-disable-next-line
+    const createdPublication = await createNewPublicationFromDoi(doiUrl, dispatch);
```

Comment:
hva er det som klages på her?

Reply:
at den ikke er brukt?

Reply:
Jepp. finnes egentlig ikke enpoint enda heller, var ment som et utgangspunkt for videre jobb